cypress/integration/BasicTermPage.feature

```diff
@@ -28,3 +28,8 @@ Feature: Basic Term Page
 		Then the page title is "fatty breast tissue"
 		Then the pronunciation text "(FA-tee brest TIH-shoo)" appears on the page
 		And the definition text "A term used to describe breast tissue that is made up of almost all fatty tissue. Fatty breast tissue does not look dense on a mammogram, which may make it easier to find tumors or other changes in the breast. Fatty breast tissue is more common in older women than in younger women. Fatty breast tissue is one of four categories used to describe a level of breast density seen on a mammogram." appears on the page
+
+	Scenario: Redirect from CDR ID
```

Comment:
Instead of adding a whole new scenario, you should modify the existing test in PrettyUrlRedirect.feature
Simply add 'And the system appends '?redirect=true' to the URL' as the last step in the mentioned above feature

Reply:
ah ok